src/applications/personalization/search-representative/config/basic-information/basicInformation.js

```diff
@@ -0,0 +1,34 @@
+export const schema = {
+  type: 'object',
+  title: 'What kind of representative are you looking for?',
+  properties: {
+    vso: {
+      type: 'boolean',
+    },
+    representativeOfVSO: {
+      type: 'boolean',
+    },
+    attorney: {
+      type: 'boolean',
+    },
+    claimsAgent: {
+      type: 'boolean',
+    },
+  },
+};
+
+export const uiSchema = {
+  'ui:description': 'Chech all that apply',
+  vso: {
+    'ui:title': 'Veteran Service Organization (VS0)',
+  },
+  representativeOfVSO: {
+    'ui:title': 'Representative in a Veteran service organization',
```

Comment:
I think Veteran service organization should be capitalized similar to vso to maintain consistency -> **V**eteran **S**ervice **O**rganization

Reply:
Derp, yep I updated this